renderer.js

```diff
@@ -121,11 +133,15 @@ function enterFullscreen () {
 
 function exitFullscreen () {
   if (isFullscreen()) return;
-  fixUpCanvas();
-  resetCamera(camera);
+  handleResize();
   safeToRenderStereo = false;
 };
 
+function handleResize() {
+  fixUpCanvas();
```

Comment:
Probably because I rebase the master commit in my master and this updates my pull request. Since i'm updating the pull request to match your suggestions I thought this was ok.